frontend/javascripts/viewer/model/sagas/saving/save_queue_draining_saga.tsx

```diff
@@ -69,72 +72,114 @@ export function* pushSaveQueueAsync(): Saga<never> {
-    //    queue is empty. Note that the save queue might be added to
-    //    while saving is in progress. Still, the save queue will be
-    //    drained until it is empty. If the user hits save and continuously
-    //    annotates further, a high number of save-requests might be sent.
-    // 2) forcePush is false
-    //    The auto-save interval was reached at time T. The following code
-    //    will determine how many items are in the save queue at this time T.
-    //    Exactly that many items will be sent to the server.
-    //    New items that might be added to the save queue during saving, will be
-    //    ignored (they will be picked up in the next iteration of this loop).
-    //    Otherwise, the risk of a high number of save-requests (see case 1)
-    //    would be present here, too (note the risk would be greater, because the
-    //    user didn't use the save button which is usually accompanied by a small pause).
-    // 3) In a live collab scenario we need to drain the whole save queue to get a state
-    //    where we are sure that server is in sync with the backend and after the saving the
-    //    annotation state can be used as a new rebase-able version (RebaseRelevantAnnotationState).
-    //    TODO: Later iterations of live collaboration might need to change this behaviour here.
-    //    e.g. continuous skeleton tracing might save too long / endlessly if traced very fast.
-    //    See https://github.com/scalableminds/webknossos/pull/8723#discussion_r2419981285
-    const currentMutexFetchingStrategy = yield* call(getCurrentMutexFetchingStrategy);
-    const isLiveCollabActive = currentMutexFetchingStrategy === MutexFetchingStrategy.AdHoc;
-
-    const itemCountToSave =
-      forcePush || isLiveCollabActive
-        ? Number.POSITIVE_INFINITY
-        : yield* select((state) => state.save.queue.length);
-    let savedItemCount = 0;
-    while (savedItemCount < itemCountToSave) {
-      saveQueue = yield* select((state) => state.save.queue);
-
-      if (saveQueue.length > 0) {
-        savedItemCount += yield* call(sendSaveRequestToServer);
-      } else {
-        break;
+    let shouldRetryOnConflict = true;
+    let retryCount = 0;
+    while (shouldRetryOnConflict) {
+      shouldRetryOnConflict = (yield* call(synchronizeAnnotationWithBackend, forcePush))
+        .shouldRetryOnConflict;
+      ++retryCount;
+      if (retryCount > MAX_ON_CONFLICT_RETRIES) {
+        const annotation = yield* select((state) => state.annotation);
+        yield* call(
+          [ErrorHandling, ErrorHandling.notify],
+          new Error("Saving annotation repeatedly failed due to conflict '409' status code"),
+          { annotationVersion: annotation.version, annotationId: annotation.annotationId },
+        );
+        Toast.error(
+          "Saving your changes failed repeatedly due to conflict with other user's changes. Please consider reloading to resolve this. This will use your latest unsaved changes.",
```

Comment:
```suggestion
          "Saving your changes failed repeatedly due to conflict with other user's changes. Please consider reloading to resolve this. This will lose your latest unsaved changes.",
```

unless I'm missing sth here?